tests/test_Store.py

```diff
@@ -0,0 +1,11 @@
+import unittest
+
+import pandas as pd
+
+from shift_detector.precalculations.Store import InsufficientDataError, Store
+
+
+class TestStore(unittest.TestCase):
+
+    def test_min_data_size_is_enforced(self):
+        self.assertRaises(InsufficientDataError, Store, df1=pd.DataFrame(), df2=pd.DataFrame([0]))
```

Comment:
You only test that the assertion is raised if there is not enough data. I think you should also test that there is no assertion raised if there is enough data.

Newline at the end of the file is missing.